themes/theme-ios11/pages/Product/components/Header/components/Price/index.jsx

```diff
@@ -17,7 +17,7 @@ import styles from './style';
  * @param {Object} props The component props.
  * @return {JSX}
  */
-const Price = ({ hasVariants, price }) => (
+const Price = ({ recalculatePrice, price }) => (
```

Comment:
Please rename this prop. `<Price price={...} recalculatePrice>` - what does that prop mean?

Reply:
done